src/helpers/format.ts

```diff
@@ -40,6 +40,33 @@ type InternalFormat = (
   | { type: "DATE"; format: string }
 )[];
 
+// TODO in the future : remove these constants MONTHS/DAYS, and use a library such as luxon to handle it
+// + possibly handle automatic translation of day/month
+const MONTHS: Readonly<Record<number, string>> = {
+  0: "January",
```

Comment:
should be translated hein mes p'tits choux :smiley_cat: 
`_lt("January")`

Reply:
https://github.com/odoo/o-spreadsheet/pull/1818